spec/frequency_classes_service_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'frequency_classes_service'
+
+describe 'FrequencyClassesService' do
+  let(:frequency_classes_service) { FrequencyClassesService.new }
+  let(:sample_word) { 'haben' }
+
+  describe '#initialize' do
+    it 'should return a FrequencyClassesService instance' do
```

Comment:
I like to use these kinds of tests to build up from 0 code following the pure TDD approach, but once you have some functionality working this type of assertion becomes unnecessarily pedantic, so I suggest deleting it.